core_debug.tex

```diff
@@ -97,7 +98,7 @@ \section{Single Step}
 \section{Reset}
 
 If the halt signal (driven by the hart's halt request bit in the Debug Module)
-is asserted when a hart comes out of reset, the hart must
+or \Fresethaltreq are asserted when a hart comes out of reset, the hart must
```

Comment:
I don't think its \Fresethaltreq. It's the conceptual {resethaltreq} bit.

Reply:
I've made \Fresethaltreq point to a description that says that.

Reply:
but that's only readable if someone is looking at a PDF, clickable version of the spec? That seems tricky to rely on.

Reply:
Ah, I see, it's ONLY that it's not actually a bit in any register. Ok.